src/core/include/openvino/op/is_inf.hpp

```diff
@@ -55,6 +55,11 @@ class OPENVINO_API IsInf : public Op {
         m_attributes = attributes;
     }
 
+    OPENVINO_SUPPRESS_DEPRECATED_START
+    bool evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const override;
```

Comment:
The same

Reply:
Will do, in progress, found some issues with tests after switch to new API